.aicoe-ci.yaml

```diff
@@ -1,15 +1,37 @@
 check:
   - thoth-build
   - thoth-precommit
-build:
-  base-image: quay.io/thoth-station/s2i-thoth-ubi8-py38:v0.20.1
-  build-stratergy: Source
-  registry: quay.io
-  registry-org: thoth-station
-  registry-project: elyra-aidevsecops-tutorial
-  registry-secret: thoth-station-thoth-pusher-secret
-deploy:
-  project-org: "thoth-station"
-  project-name: "elyra-aidevsecops-tutorial"
-  image-name: "elyra-aidevsecops-tutorial"
-  overlay-contextpath: "manifests/overlays/test/imagestreamtag.yaml"
+  - overlays-build
+overlays_dir: overlays
+overlays:
+  - name: download-dataset
+    build:
+      base-image: quay.io/thoth-station/s2i-thoth-ubi8-py38:v0.20.1
+      build-stratergy: Source
+      registry: quay.io
+      registry-org: thoth-station
+      registry-project: elyra-aidevsecops-dataset
+      registry-secret: thoth-station-thoth-pusher-secret
+
+  - name: test-model
```

Comment:
added missing overlay for `training`, we don't need test_model image for now. and I need to be sure also the overlays have correct naming withou `_` right?

Reply:
yes thats correct, for now  i have added that change to this PR.